localstack/services/dynamodb/provider.py

```diff
@@ -1269,6 +1269,18 @@ def should_throttle(self, action):
 # ---
 # Misc. util functions
 # ---
+def _get_size_bytes(item) -> int:
+    def _decode_dict(_item):
+        for k, v in _item.items():
+            if isinstance(v, bytes):
+                v = v.decode()
+                _item.update({k: v})
+
+    try:
+        size_bytes = len(json.dumps(item))
```

Comment:
As mentioned below, decoding the bytes may not always work - perhaps we should simply check if JSON dumping is possible, otherwise return the length of the string representation of the item? (which should be a good-enough approximation of its length)

```
try:
    return len(json.dumps(item))
except TypeError:
    return len(str(item))
```

Reply:
The utility `_decode_dict` seemed to work also with the example below. However you are right; we might still encounter some errors while decoding and the length of the string is a fair approximation 👍